src/pure/message.ml

```diff
@@ -39,7 +36,7 @@ type 'a message = {
   mutable headers : (string * string) list;
   mutable client_stream : Stream.stream;
   mutable server_stream : Stream.stream;
-  mutable body : string promise option;
+  mutable body : string option;
```

Comment:
Before this change, `message.body` represented a potentially delayed computation for a complete body. As I understand it, after this change, the body is now either absent, or present, but it cannot be delayed. Should this be a `(unit -> string) option` or some kind of Eio promise option now?